build.sh

```diff
@@ -75,6 +76,7 @@ function usage()
     echo "                                 will clean all the generated code and binaries "
     echo "    --debug                      Compile against the RTI Connext Debug          "
     echo "                                 libraries. Default is against release ones.    "
+    echo "    --build-doc                  Generate the HTML and PDF documentation.       "
```

Comment:
@javmorcas should we have it as internal. I mean not say anything about it in the `--help`

Reply:
I see that potentially this might be used for some of our customers.

Reply:
sure